Library/Homebrew/test/.rubocop_todo.yml

```diff
@@ -200,6 +200,7 @@ RSpec/VerifiedDoubles:
     - 'formula_spec.rb'
     - 'language/python_spec.rb'
     - 'linkage_cache_store_spec.rb'
+    - 'livecheck/livecheck_spec.rb'
```

Comment:
Would be good to use these if possible but no big deal.

Reply:
The reason I've not used a verified double is because I'm using the double to stand in for parsed arguments. If it has to be made verified, I'll have to define (?) a class the same way the arguments parser does, which I feel may be too much for a test. What do you think @MikeMcQuaid?

Reply:
Fine with me 👍🏻